### PR TITLE
add creatimeTimestamp to entities (kt)

### PR DESCRIPTION
**Pull request by mmandlis**

### CODE REVIEW

File: java/arcs/android/crdt/ParcelableRawEntity.kt [hunk 1]
```diff
@@ -67,10 +68,13 @@ data class ParcelableRawEntity(
                 collections[key] = set
             }
 
+            @Suppress("GoodTime") // use Instant
```
Comment (galganif) [inline java/arcs/android/crdt/ParcelableRawEntity.kt]: why not instant? is the comment a todo (in that case can it be rephrased as a todo?)?

Comment (mmandlis) [reply java/arcs/android/crdt/ParcelableRawEntity.kt]: iirc it's not cross platform, so i couldn't add it. this is an internal check, so we're just suppressing the error for now.

**Comment by mmandlis** [comment]
> Would you be able to add support for this to the DatabaseImpl as well?

is it ok, if i do it in a follow up PR?

File: java/arcs/android/crdt/ParcelableRawEntity.kt (hunk 1 above)
Comment (galganif) [reply java/arcs/android/crdt/ParcelableRawEntity.kt]: can you say that as a comment in the code ("we are not using instant because ....")

**Comment by jasonwyatt** [comment]
Yeah, that's fine

**Pull request merged**